apps/meteor/tests/unit/server/users/saveUserIdentity.spec.ts

```diff
@@ -131,4 +132,17 @@ describe('Users - saveUserIdentity', () => {
 		expect(stubs.updateUserReferences.called).to.be.true;
 		expect(result).to.be.true;
 	});
+
+	it('should update bio to an empty string and call setBio', async () => {
+		stubs.findOneUserById.returns({ bio: 'Some bio text', username: 'oldUsername' });
+		stubs.setRealName.returns(true);
+		stubs.setUsername.returns(true);
+		stubs.setBio = sinon.stub();
+		const result = await saveUserIdentity({ _id: 'valid_id', bio: '' });
+		expect(stubs.setRealName.called).to.be.false;
+		expect(stubs.setUsername.called).to.be.false;
+		expect(stubs.updateUsernameOfEditByUserId.called).to.be.false;
+		expect(stubs.setBio.calledWith('valid_id', '')).to.be.true;
```

Comment:
Hi @Gustrb ,

I tried running the tests using yarn testunit, but I wasn’t able to reproduce the failure for the above test. Could you please guide me on the steps or the specific command you used to encounter this failure? If there’s a different setup or additional dependencies required, let me know so I can address the issue effectively.

Thanks in advance for your help.

Reply:
Hey, it is failing on the CI, to run this specific test locally run
```bash
$ yarn .testunit:server -g saveUserIdentity
```

Reply:
Ok, thanks

Reply:
Hi @Gustrb,

I wanted to clarify a small confusion I had earlier. Initially, I was writing tests for the saveUserProfile method in a file that was intended for testing the saveUserIdentity method. However, after reviewing the test structure, I realized that there were no existing tests for the saveUserProfile method. As a result, I created a new test file specifically for it.

Although I added the new test file path to the .mocharc.js configuration so that it would be included in the test execution, I'm still facing an issue where the test is not being picked up during execution. I’m unable to run the test file, and I was hoping you could guide me on how to resolve this issue.

Could you please let me know the correct approach to ensure that this test file is included and executed?

Thanks in advance.

Reply:
Hey, I am pretty happy with the API test, from my side there is no need to add unit tests
Also, there is no need to add that path into .mocharc

Reply:
Thanks for the clarification, gustrb! Just to confirm, is it necessary to have a unit test specifically for the saveUserProfile method, or is the API test sufficient to cover this functionality?

If no unit test is needed, should I go ahead and create a new commit to remove the changes related to the test file and .mocharc.js adjustments?

Let me know how you’d prefer me to proceed!

Reply:
the API test is sufficient :)
I would remove those changes

Reply:
it will be probably tested by someone else so we can get the QA tested tag
I'll approve once you change that, but there might be more reviews from other people

Reply:
> the API test is sufficient :) I would remove those changes

Ok, but I kindly request that you remove these changes( the saveUserProfile file and the path in mocharc.js file) so that it does not cause any error.

Reply:
I can't do that right now, I thought you were working on that(?)
By I would I mean, "if I were you" :)

Reply:
Now that I understand the expectations better, I have removed the unintended parts of the code, including the unnecessary test file and its associated configuration. The remaining changes are ready for further testing.